lib/application.js

```diff
@@ -618,6 +618,34 @@ app.listen = function listen() {
   return server.listen.apply(server, arguments);
 };
 
+/**
+ * Creates a router that inherits options from app.settings
+ * and allows them to be overriden by passing in an options
+ * object
+ *
+ * @param {Object} [options]
+ * @return {router}
+ * @public
+ */
+app.createRouter = function (options) {
+  var opts = {
+    strict: this.enabled('case sensitive routing'),
```

Comment:
I think the `case sensitive routing` is a typo on this line. Let's fix that and add a test that will actually catch that mistake in the future 👍

Reply:
After starting at a screen for this many hours eventually you go blind 😆

Reply:
That's why we review code 😄 happens to everyone